letsencrypt/cli.py

```diff
@@ -636,8 +638,12 @@ def preprocess_args(self, args):
                 self.verb = token
                 return reordered
 
-        self.verb = "run"
-        return args + ["run"]
+        if "--standalone" in args and "--installer" not in args and "-i" not in args:
```

Comment:
I don't think we should be changing the verb based on the plugin flags given. `run` is the default except for when it's not? That's confusing.

I think the behavior without these 6 lines of code is acceptable. If you have an installer available, it'll be used with `standalone` in run (this actually works if your server isn't currently running). If you don't, you'll get one of the messages like "No installers are available on your OS yet; try running "letsencrypt-auto auth" to get a cert you can install manually" which I think is perfectly acceptable and even preferred behavior.